src/applications/Applications.ExtractImages/ExtractImagesHost.cs

```diff
@@ -44,9 +44,9 @@ public class ExtractImagesHost : MicroserviceHost
 
             _fileSystem = fileSystem ?? new FileSystem();
 
-            string extractRoot = Globals.FileSystemOptions.ExtractRoot;
-            if (!_fileSystem.Directory.Exists(extractRoot))
-                throw new DirectoryNotFoundException($"Could not find the extraction root '{extractRoot}'");
+            string extractionRoot = Globals.FileSystemOptions.ExtractRoot;
```

Comment:
In future please can we try and avoid renaming variables without good reason since it clutters the PR and makes it harder to review

Reply:
Agreed - have reverted this.